### PR TITLE
Sync definitions to oss bucket, which acts as a capability center

### LINKED ISSUE #1702: Sync Definition files to Cloud Storage and make it as default Registry
1. sync definitions files in this repo to Alibaba Cloud OSS just like our charts. @zzxwill 
2. Configure `registry.kubevela.net` as the OSS custom domain.
3. Make `registry.kubevela.net` as the default regisrty

Bucket name: kubevela-registry  
kubevela-registry.oss-cn-beijing.aliyuncs.com

### PR DESCRIPTION
**Pull request by zzxwill**
Fix #1702

### CODE REVIEW

File: .github/workflows/registry.yml [hunk 1]
```diff
@@ -128,3 +128,33 @@ jobs:
...
+      - name: Get git revision
+        id: vars
+        shell: bash
+        run: |
+          echo "::set-output name=git_revision::$(git rev-parse --short HEAD)"
+      - uses: oprypin/find-latest-tag@v1
+        with:
+          repository: oam-dev/kubevela
+          releases-only: true
+        id: latest_tag
```
Comment (wonderflow) [inline .github/workflows/registry.yml:149]: where is it used ？

File: .github/workflows/registry.yml [hunk 2]
```diff
@@ -128,3 +128,33 @@ jobs:
...
+      CAPABILITY_DIR: capabilities
+      CAPABILITY_ENDPOINT: oss-cn-beijing.aliyuncs.com
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Get git revision
+        id: vars
+        shell: bash
+        run: |
+          echo "::set-output name=git_revision::$(git rev-parse --short HEAD)"
```
Comment (wonderflow) [inline .github/workflows/registry.yml:144]: where do we used?

**Review by wonderflow — APPROVED** [review]
please remove the unused steps

File: .github/workflows/registry.yml (hunk 2 above)
Comment (zzxwill) [reply .github/workflows/registry.yml]: Yes, this should be removed. Thanks.

**Pull request merged**